index.html

```diff
@@ -616,7 +616,7 @@ <h3>Proofs</h3>
 and local calendar preferences of the individual [[?LTLI]]. Conversion of
 timestamps to local time values are expected to consider the time zone
 expectations of the individual. See
-<a data-cite="VC-DATA-MODEL-2.0#representing-time"></a> for more details about
+<a data-cite="?VC-DATA-MODEL-2.0#representing-time"></a> for more details about
```

Comment:
I haven't checked, but Respec should be smart enough to notice that this is inside a Note and so put it in the informative references section even if you don't add the `?`. If it's not doing that, we should file a bug.

Reply:
Yes, I thought the same and was surprised when the informative reference jumped to the normative references in the appendix. At that point, I scanned through the document and manually set all references to informative (that were supposed to be informative references)... and the problem went away.

I didn't then go back and try to reproduce it by removing the "?" characters from `data-cite`. I guess I'll try that now as I have the spec up in another window...

Hmm, so, it worked... I tried two things:

1. Set the above manually in `data-cite` and it moved it from normative to informative.
2. Set the section as `class="informative"` and it moved it from normative to informative.

So, ReSpec seems to be operating as expected. I think the issue might be that the appendix is marked as normative, when it should be informative... so, I'll fix that when I go back through and clean up this PR.

Thanks for the nudge to track down the issue @jyasskin.